README.md

```diff
@@ -0,0 +1,28 @@
+# azure_lightning_flask
+A [Flask](http://flask.pocoo.org/) server to serve single-page applications deployed using the [lightning strategy](http://ember-cli-deploy.com/docs/v1.0.x/the-lightning-strategy/) on [Azure Table Storage](https://azure.microsoft.com/en-us/services/storage/tables/). Designed to work with [Ember-cli-deploy-azure](https://github.com/duizendnegen/ember-cli-deploy-azure), but should be compatible with any lightning strategy demployment on Azure Table Storage via configuration.
+
+## Installation
+```
+pip install -r requirements.txt
+```
+## Running
+```
+export FLASK_APP=runserver.py
+flask run
```

Comment:
I assume venv would be preferred, but that's up to whoever will be running this in prod environments.

Reply:
WILL NOT DO